libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/ParserUtil.java

```diff
@@ -55,6 +55,7 @@ public static GeneralSqlType getGeneralSqlType(BasicResult result) {
             case DROP:
             case TRUNCATE:
             case ALTER:
+            case COMMENT:
```

Comment:
What about CALL and SET_SESSION?

Reply:
The call statement is used to invoke procedures and functions

Reply:
The `SET SESSION` statement belongs to the syntax of mysql. For example, SET SESSION time_zone = '+00:00';

Reply:
call and set_session belong to unknow